cpp/src/slice2cpp/Gen.cpp

```diff
@@ -3332,16 +3277,27 @@ Slice::Gen::InterfaceVisitor::visitInterfaceDefEnd(const InterfaceDefPtr& p)
           << getUnqualified("::Ice::Current&", scope) << " current)";
         C << sb;
 
-        C << nl << "::std::pair<const ::std::string*, const ::std::string*> r = "
-          << "::std::equal_range(" << flatName << ", " << flatName << " + " << allOpNames.size()
+        C << sp;
+        C << nl << "static constexpr ::std::string_view allOperations[] = ";
```

Comment:
It's nicer to use a constexpr string_view since it gets initialized at a compile-time. Lots of statics increase start-up time.